### PR TITLE
Update datasets

### PR DESCRIPTION
**Pull request by blinkseb**
- Include Run2016H datasets
  - Update JSON file to the latest one
  - Use rereco for 2016B-G
  - Add new TT sample with new tune (unfortunately less statistics than the old sample and no fully leptonic sample)
  - Remove inexistent TT sample
  - Update HH analysis to include also inclusive TT samples

### CODE REVIEW

**Comment by blinkseb** [comment]
Updated DY NLO sample (new one has ~3x more stats)

**Comment by blinkseb** [comment]
Global tags added for data samples

**Comment by blinkseb** [comment]
It seems like `Run2016H-PromptReco-v1` contains no certified runs. I'll remove it from the list

**Comment by BrieucF** [comment]
Is it annoying to leave it there for the moment? There are still JSON updates foreseen and they sometimes recover runs.

**Comment by blinkseb** [comment]
crab complains there's no job for this task and leave an empty folder, but otherwise no issue.

File: datasets/data_DoubleEG.json [hunk 1]
```diff
@@ -1,49 +1,82 @@
...
+            "globalTag": "80X_dataRun2_2016SeptRepro_v4",
+            "certified_lumi_file": "https://cms-service-dqm.web.cern.ch/cms-service-dqm/CAF/certification/Collisions16/13TeV/Cert_271036-284044_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt"
         },
 
-        "/DoubleEG/Run2016G-PromptReco-v1/MINIAOD": {
-            "name": "DoubleEG_Run2016G-PromptReco-v1",
+        "/DoubleEG/Run2016G-23Sep2016-v1/MINIAOD": {
+            "name": "DoubleEG_Run2016G-23Sep2016-v1",
             "units_per_job": 200,
             "run_range": [278820, 999999],
```
Comment (OlivierBondu) [inline datasets/data_DoubleEG.json]: could you update the end run for Run2016G ?

File: datasets/data_MuonEG.json [hunk 2]
```diff
@@ -1,51 +1,84 @@
...
+            "globalTag": "80X_dataRun2_2016SeptRepro_v4",
+            "certified_lumi_file": "https://cms-service-dqm.web.cern.ch/cms-service-dqm/CAF/certification/Collisions16/13TeV/Cert_271036-284044_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt"
         },
 
-        "/MuonEG/Run2016G-PromptReco-v1/MINIAOD": {
-            "name": "MuonEG_Run2016G-PromptReco-v1",
+        "/MuonEG/Run2016G-23Sep2016-v1/MINIAOD": {
+            "name": "MuonEG_Run2016G-23Sep2016-v1",
             "units_per_job": 200,
             "run_range": [278820, 999999],
```
Comment (OlivierBondu) [inline datasets/data_MuonEG.json]: same here

**Review by OlivierBondu — CHANGES_REQUESTED** [review]
Thanks ! looking good except two small corrections !

**Comment by blinkseb** [comment]
End run for 2016G updated. I also added a new commit updating HT binned DY samples

**Review by OlivierBondu — APPROVED** [review]

**Comment by OlivierBondu** [comment]
thanks!

**Pull request merged**